<?xml version="1.0"?>
<launch>

    	<arg name="uav1_gcs2" default="1" />
	<arg name="gcs_uavip" default="192.168.50.199" />
	<arg name="uav_video_topic" default="/kcfTracker/image_raw" />
        <!-- 
                [ /usb_cam/image_raw] 		usb_cam
		[ /usb_cam/image_rect_color] 	usb_cam   
                [ /camera/color/image_raw] 	/realsense2_camera
                [ /camera/color/image_raw] 	/csi_cam_0
        -->

	<param name="uav1_gcs2" value="$(arg uav1_gcs2)" type="int" />
	<param name="gcs_uavip" value="$(arg gcs_uavip)" type="string" />
	<param name="uav_video_topic" value="$(arg uav_video_topic)" type="string" />

        <node pkg="map_widget" type="map_widget" name="map_widget_uav" output="screen" required="true">
        </node>

</launch>

